docs/introduction/QuickStart.mdx

```diff
@@ -0,0 +1,177 @@
+---
+id: quick-start
+title: Quick Start
+sidebar_label: Quick Start
+hide_title: true
+---
+
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+
+import { DetailedExplanation } from '../components/DetailedExplanation'
+import { Note } from '../components/Note'
+
+# Quick Start
+
+<Note>
+
+### What You'll Learn
+
+- What Redux is, and why you should use it
+- How Redux relates to React
+- How to set up and start using Redux quickly
+- What a small working React+Redux app looks like
+
+### Prerequisites
+
+There are several things you should know in advance before learning Redux:
+
+- Basic familiarity with HTML & CSS, and the DOM.
+- Basic knowledge of JavaScript and programming.
+- Familiarity with ES6 syntax and features.
+- Basic knowledge of React, including components, props, and state.
+- Node.js and `npm` installed globally.
+
+</Note>
+
+## What Is Redux?
+
+**Redux** is a predictable state container for JavaScript apps. It acts as the data layer inside an application.
+
+It helps you write applications that behave consistently and can run in different environments (client, server, and native).
+It also helps you to write code that is testable, and enables you to better understand when, where, why, and how the data in your
+application has changed.
+
+Redux does this by asking you to:
+
+- Place some data (**_"state"_**) into a special object called a **_"store"_**
+- Write separate functions called **_"reducers"_** that know how to update the state inside the store
+- Follow specific rules for how your code can update and read the state
+
+### What does Redux include?
+
+The core Redux library consists of a few API functions. The most important API is the `createStore` function.
+
+<Note type="error">
+
+**TODO** figure out what else to put here, or if this should be dropped
+
+</Note>
```

Comment:
Personally I don't think this section is needed in a quick start, since it's specifically for getting something quickly rather than overviewing the whole API, which I think the rest of the docs introduce well enough